server/tweetSearch.js

```diff
@@ -38,6 +39,18 @@ module.exports = function(client, fs, eventConfigFile, mkdirp) {
         if (tweets.length === 0) {
             return;
         }
+        if (inApprovalMode && tag !== "official") {
+            tweets.forEach(function(tweet) {
```

Comment:
Yeah that would tie in nicely with my other comment, I didn't call setDeletedStatus because it does some checking on the tweet store and the tweets weren't in the tweet store yet.